src/utils/query-params.ts

```diff
@@ -0,0 +1,15 @@
+export class QueryParams {
+  readonly params;
+  constructor(query) {
+    this.params = query;
+  }
+  get q() {
+    return this.params.q;
+  }
```

Comment:
что такое q? Нужно более понятное название переменной

Reply:
В текущей реализации `q` будет использоваться для названия параметра в строке url. По крайней мере до конечного решения по названию всех параметров.
При этом сама переменная, используемая в коде, называет понятно `search`.
```suggestion
enum enumParamsList {
  search = 'q',
  sort = 's',
}
enum enumParamsSingle {
  id = 'id',
}
```